5주차/[Lv.2]_스킬트리/주다현.js

```diff
@@ -0,0 +1,15 @@
+function solution(skill, skill_trees) {
+  let answer = 0;
+
+  skill_trees = skill_trees.map((v) => {
+    return v
+      .split('')
+      .filter((x) => skill.includes(x))
+      .join('');
+  });
+
+  skill_trees.forEach((v) => {
+    if (skill.includes(v) && skill.indexOf(v) == 0) answer++;
```

Comment:
오.. 👍👍👍 검사하려는 스킬이 아닌 문자를 모두 제외했으니 indexOf 로 스킬트리 문자에 해당되는지만 체크하면 되는군요
입력 데이터만 잘 정제한다면 이렇게 간단하게 해결할 수 있었군요 대박입니다
추가적으로 조건 하나는 제외해도 괜찮을 것 같아요!
```suggestion
    if (skill.indexOf(v) === 0) answer++;
```